packages/p2p-v2/src/pages/my-profile/screens/MyProfile/MyProfile.tsx

```diff
@@ -14,13 +16,30 @@ const TABS = ['Stats', 'Payment methods', 'Ad details', 'My counterparties'];
 const MyProfile = () => {
     const { isMobile } = useDevice();
     const { queryString, setQueryString } = useQueryString();
+    const { data } = usePoiPoaStatus();
+    const { data: advertiserStats, failureReason, isLoading } = useAdvertiserStats();
+    const { isPoaVerified, isPoiVerified } = data || {};
+    const [isNicknameModalOpen, setIsNicknameModalOpen] = useState<boolean | undefined>(false);
 
     const currentTab = queryString.get('tab');
 
+    useEffect(() => {
+        if (isPoaVerified && isPoiVerified && !!failureReason) setIsNicknameModalOpen(true);
```

Comment:
we need to check first if poa is required in the country then we can check if it is verified

Reply:
ahhhh ok, will amend it 👍